src/web/js/trove/chart-lib.js

```diff
@@ -781,7 +1167,7 @@ ${labelRow}`;
       }
 
       function setup(restarter) {
-        const tmp = f(globalOptions, rawData);
+        var tmp = f(globalOptions, rawData);
```

Comment:
Is this an important `const` -> `var` change?

Reply:
It doesn't seem to be, as far as I can tell the only things that we really edit are the variables of tmp which can still be changed if tmp is a const and tmp doesn't every really seem to be reassigned.